### DISCUSSION TITLE
Super Game Boy 1/2 bootrom dumping support

### DISCUSSION THREAD

**Discussion by DrAzathoth** [sanni/cartreader#905] [Ideas & Suggestions]
Is it possible to add support for dumping the SGB1/2 bootrom (BIOS)? I understand this is fairly unnecessary as it doesn’t have a splash screen and most emulators that support the SGB just HLE the bootrom. Personally, I would love to dump it for archival purposes and more accurate emulation.

**Comment by 7ank0v1c** [comment]
Personally I would like to dump this too (no idea if it is possible though) (also assuming this is different to the rom dumped already?)

**Reply by DrAzathoth** [reply]
The bootroms have been dumped. The only real difference between the SGB and SGB2 is that there is a feature that allows games to distinguish between the two (like the two different borders for Tetris DX).

https://gbdev.gg8.se/files/roms/bootroms/readme.txt

Edited link to remove downloads to the actual bootroms themselves.

**Reply by Arctic-Circle-System** [reply]
There are some prototype SGBs floating around though, most notably one with SGB-CPU X1 that hasn't been dumped. Though I am not sure how one would go about dumping it. Well, aside from [this.](https://www.its.caltech.edu/~costis/sgb_hack/)
